packages/react/src/middleware/default-organization.ts

```diff
@@ -0,0 +1,13 @@
+import { OrganizationDetails, User } from "@slashid/slashid";
+import { LoginMiddleware } from "../domain/types";
+
+export const defaultOrganization = (oid: string | ((organizations: OrganizationDetails[]) => string)): LoginMiddleware => async ({ user }) => {
```

Comment:
This looks good! Maybe consider adding some inline docs in TS Doc format so that pops up in the editor?

Reply:
done